### ISSUE TITLE
https://crafatar.com/ is not working.

### ISSUE THREAD

**Issue by MrPiva** [crafatar/crafatar#257]
The website https://crafatar.com/ changed, now it shows something of "Car insurance"

**Comment by fjeddy** [comment]
Wtf! I'm hoping this is just some server-switch / default host config issue and not Crafatar shutting down :/

**Comment by jomo** [comment]
No worries, Crafatar is not shutting down and certainly not without prior notice!

The issue has been fixed and it shouldn't happen again.

---

Details: We recently switched from nginx to HAProxy (see #256) and the logrotate config for haproxy wasn't working properly. As a result, the disk ran out of space which caused A) the app to crash, which wouldn't be a big issue because we forward requests to a backup server, if not B) our DNS cache also failed to update the IP address of the backup server (which is running on Heroku and they change their Amazon EC2 IP addresses frequently), so effectively all requests were sent to a recycled IP address (which happens to be used by a car insurance company now).

As a bonus, I had added the `ssl` and `ca-file` options in the haproxy config but forgot `verify` and `verifyhost`, so the SSL validation error this obviously caused was ignored ¯\\\_(ツ)\_/¯

I have fixed both logrotate and haproxy configs.
